torchbenchmark/models/Background_Matting/__init__.py

```diff
@@ -82,18 +80,18 @@ def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
         netG.apply(conv_init)
         self.netG = netG
 
-        if self.device == 'cuda':
-            self.netG.cuda()
+        self.netG.to(self.device)
+        # if self.device == 'cuda':
             # TODO(asuhan): is this needed?
-            torch.backends.cudnn.benchmark = True
+            # (whc) - i'm removing this so lazy/cuda have an even playing field
+            # torch.backends.cudnn.benchmark = True
```

Comment:
@wconstab by default we set `cudnn.benchmark` to `False`. Some models set it to `True` instead, which is totally fine (since we are now running each test in separate process), but we need to update the metadata file in `https://github.com/pytorch/benchmark/blob/main/torchbenchmark/models/demucs/metadata.yaml#L5`.